Makefile

```diff
@@ -68,17 +83,27 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: container-unit-test
+container-unit-test: ## Build docker image with the manager and execute unit tests.
+	docker build -f Dockerfile --label $(IMAGE_TAG_BASE)-$@:$(VERSION)-$@ -t $(IMAGE_TAG_BASE)-$@:$(VERSION) --load --target testing .
+	docker run --rm -t --name $@-dws-test-driver  $(IMAGE_TAG_BASE)-$@:$(VERSION)
+
 # If you wish built the manager image targeting other platforms you can use the --platform flag.
 # (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build --load -t ${IMG} .
```

Comment:
See my other comment about `--load`.

Reply:
If you store it as an image tarball here, rather than putting into the local image cache, then that would not match up with what we want when we run the `docker-push` or `kind-push` targets.
It also means that `container-unit-test` (after you remove the --load from there) cannot take advantage of cached layers in the image cache.